### ISSUE TITLE
Correct way to alter messagelist ordering of subject, fromto and date|size

### ISSUE THREAD

**Issue by lachlan-00** [roundcube/roundcubemail#9409]
Hi,

I've changed the generated CSS to order the subject on top which is how i'd prefer it.

![image](https://github.com/roundcube/roundcubemail/assets/1305249/d5e43519-4422-4c96-872f-ffee3ee43ac1)

I did this by just updating the CSS with an order as i can't see these details in mail.html

```
.messagelist td.subject span.subject {
  order: 1;
}

.messagelist td.subject span.date, .messagelist td.subject span.size {
  order: 3;
}

.messagelist td.subject span.fromto {
  order: 2;
}

```

I'm trying to work out where this is set, the template calls roundcube objects in the template here.
https://github.com/roundcube/roundcubemail/blob/d9d51b8d9c8bdab60c2b994bd06d75a3e75f2767/skins/elastic/templates/mail.html#L104

Where is that object created? (Possibly `/program/actions/mail`?)

I tried updating `skins/elastic/styles/styles.less` but i'm unsure how that works compared to regular css.

So TLDR is this something that can be configured or should i just keep my css changes?

Thanks for taking the time to read through this, the bold text on the bottom was really irking me.

**Comment by alecpl** [comment]
It is not configurable. Maybe in javascript you could work around `widescreen_list_template` use and/or utilize the `insertrow` event. You still will need your css styles.

ps. less files are used to build css, the process is described in the skin readme file.

**Comment by lachlan-00** [comment]
no problem, thanks for responding.
